Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM alpine:3.11
+
+RUN apk add curl
+
+RUN apk add python3
+
+RUN curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python3 -
+
+COPY . .
+
+RUN pip3 install flask
+
+RUN source $HOME/.poetry/env && poetry update && poetry install
+
+RUN pip3 install gunicorn
+
+RUN pip3 install requests
+
+FROM base as production
+ENV FLASK_ENV=production
+ENTRYPOINT ["gunicorn", "app:app"]
+CMD ["--config gunicorn.conf.py"]
+
+FROM base as development
+RUN poetry install
```

Comment:
Hasn't this already been done on line 13? In addition, I get a `poetry not found` error as you haven't added poetry to the path

Reply:
I think it wasn't working because I hadn't added "as base" in the first line